components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,120 @@
+import { Page } from '@/components/Page/Page'
```

Comment:
can u abstract all the mapping and details out to another file, i would ideally just want components to be rendered here